mixer/pkg/il/interpreter/interpreterRun.go

```diff
@@ -1148,6 +1150,502 @@ func (in *Interpreter) run(fn *il.Function, bag attribute.Bag, step bool) (Resul
 			opstack[sp] = t3
 			sp++
 
+		case il.LtS:
```

Comment:
This file is generared from interpreterRun.got. It doesn't seem like you have that in your PR.

Reply:
Thanks!
Good point, reading helps.